src/Control/Lens/Fold.hs

```diff
@@ -424,8 +424,8 @@ takingWhile p l pafb = fmap runMagma . traverse (cosieve pafb) . runTakingWhile
 --
 -- Note: Many uses of this combinator will yield something that meets the types, but not the laws of a valid
 -- 'Traversal' or 'IndexedTraversal'. The 'Traversal' and 'IndexedTraversal' laws are only satisfied if the
--- new values you assign also pass the predicate! Otherwise subsequent traversals will visit fewer elements
--- and 'Traversal' fusion is not sound.
+-- new value you assign to the first target also does not pass the predicate! Otherwise subsequent traversals
```

Comment:
`dropping 1 . droppingWhile p $ t` should always be lawful for any traversal `t` and any predicate `p`. Eg:

    t  = droppingWhile (<= 1) traverse
    t' = dropping 1 t

    -- over t f . over t g ≠ over t (f . g)
    >>> [1,2,3] & t .~ 0 & t .~ 1
    [1,0,0]
    >>> [1,2,3] & t .~ 1
    [1,1,1]

    -- t' on the other hand behaves lawful
    >>> [1,2,3] & t' .~ 0 & t' .~ 1
    [1,2,1]
    >>> [1,2,3] & t' .~ 1
    [1,2,1]